wireguard-go-rust-wrapper/wireguard-go/platform_specific_windows.go

```diff
@@ -12,8 +14,39 @@ import (
 	"crypto/sha256"
 
 	"github.com/google/uuid"
+	"golang.org/x/sys/windows/registry"
 )
 
+func CheckIfAdapterExists(guid string) bool {
+	regKey, err := registry.OpenKey(registry.LOCAL_MACHINE, `SYSTEM\CurrentControlSet\Control\Class\{4d36e972-e325-11ce-bfc1-08002be10318}`, registry.ALL_ACCESS)
```

Comment:
Is that key (`d36e972-e325-11ce-bfc1-08002be10318`) universal on all Win versions/computers?